fusion-plugin-font-loader-react/package.json

```diff
@@ -26,6 +26,7 @@
     "./dist/browser.es2015.es.js": "./dist/browser.es2017.es.js"
   },
   "dependencies": {
+    "fusion-react": "0.0.0-monorepo",
```

Comment:
I think you originally had this as a peerDep, any reason for moving it to deps?

Reply:
Yeah, I added it because the src code now has a direct dependence on `useService` in fusion-react